examples/tree_2d_dgsem/elixir_euler_colliding_flow.jl

```diff
@@ -0,0 +1,110 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+gamma = 1.001 # almost isothermal when gamma reaches 1
+equations = CompressibleEulerEquations2D(gamma)
+
+# This is a hand made colliding flow setup without reference. Features Mach=70 inflow from both
+# sides, with relative low temperature, such that pressure keeps relatively small
+# Computed with gamma close to 1, to simulate isothermal gas
+function initial_condition_colliding_flow_astro(x, t, equations::CompressibleEulerEquations2D)
+  # change discontinuity to tanh
+  # resolution 128^2 elements (refined close to the interface) and N=3 (total of 512^2 DOF)
+  # domain size is [-64,+64]^2
+  @unpack gamma = equations 
+  rho = 0.0247
+  c = 0.2
+  p = c*c/gamma*rho
+  vel = 13.907432274789372 
```

Comment:
Where did you get this constant from? Might be worth a comment or so...

Reply:
I'll add a comment: there is nothing fancy about it, just trying to get as close as possible to the astro setups (they have their physical units in parsec, mega years, etc...)